.github/workflows/tests.yml

```diff
@@ -26,6 +26,8 @@ jobs:
         steps:
             - name: Checkout repository
               uses: actions/checkout@v2
+              with:
+                  fetch-depth: 0
```

Comment:
We can make this a 100 or some more reasonable amount, so this is just how many commits deep should the repo be checked out. I wouldn't expect anything more than 50 or a 100 commits are realistic for a PR.